JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -131,20 +137,45 @@ fun ArticleScreen(
         topBar = {
             InsetAwareTopAppBar(
                 title = {
-                    Text(
-                        text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography.subtitle2,
-                        color = LocalContentColor.current
-                    )
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .wrapContentWidth(align = Alignment.CenterHorizontally)
+                            .padding(start = 30.dp)
+                    ) {
+                        Surface(
+                            color = Color(0xff073042),
+                            shape = CircleShape,
+                            modifier = Modifier.size(36.dp, 36.dp)
+                        ) {
+                            Image(
+                                painter = painterResource(R.drawable.ic_article_icon),
+                                contentDescription = "",
+                                colorFilter = ColorFilter.tint(Color(0xff32dd84)),
+                                modifier = Modifier.padding(4.dp)
+                            )
+                        }
```

Comment:
Can you create an drawable xml for the whole icon, including the background, and set the colors inside that drawable xml? That makes this code a lot more readable and less magical.

Reply:
I can help with creating this xml if you want!

Reply:
I created icon_article_background, let me know if you feel it is the correct implementation

Reply:
The Android logo is not centered in the circle, I checked the mocks and it should be smaller.

The mock's logo has weird antennae though.

Reply:
![image](https://user-images.githubusercontent.com/609125/130615461-b54bf4ce-b106-4e99-a954-eb3349df7ae6.png)
vs
![image](https://user-images.githubusercontent.com/609125/130615490-6fbecfae-73f2-498e-838b-9b7890dd38d4.png)